src/EditorFeatures/Test2/project.json

```diff
@@ -1,19 +1,21 @@
 ﻿{
   "dependencies": {
-    "System.Threading.Tasks.Dataflow": "4.5.24",
-    "Microsoft.VisualStudio.Composition": "14.2.19-pre",
-    "RoslynDependencies.Microsoft.VisualStudio.Platform.VSEditor": "14.3.25407",
+    "Microsoft.Tpl.Dataflow": "4.5.24",
```

Comment:
Shouldn't this be the custom "System.Threading.Tasks.Dataflow": "4.5.24" package?